javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointFeatures.qll

```diff
@@ -8,6 +8,12 @@ import javascript
 import CodeToFeatures
 import EndpointScoring
 
+/** Maximum number of descendants of an AST node to be considered to be in the "neighborhood" of that node */
```

Comment:
Before running the dev pipeline, do we want to produce several different features with different values of `maxNumDescendants`, so we can experiment and see which give good signal?

Reply:
That sounds sensible yes.